index.html

```diff
@@ -85,6 +85,8 @@ <h2>Heading</h2>
 
       <footer>
         <p>&copy; Company 2015</p>
+
+        <a href="https://webaim.org/techniques/alttext/">WebAIM Accessibility Guide</a>
```

Comment:
I would suggest adding the id tag of "basics" to the end of this URL to send the User directly to the "Alternative Text Basics" section.
```suggestion
        <a href="https://webaim.org/techniques/alttext/#basics">WebAIM Accessibility Guide</a>
```